src/main/java/com/bunq/tinker/libs/BunqLib.java

```diff
@@ -180,6 +178,17 @@ public User getUser() {
 
     public List<Card> getAllCard() {
         return getAllCard(DEFAULT_FETCH_COUNT);
+  /**
+   */
+  private void handleForbiddenException(ForbiddenException forbiddenException) {
+    if (
+        ApiEnvironmentType.SANDBOX.equals(this.environmentType) &&
+            Pattern.matches(REGEX_INSUFFICIENT_AUTHENTICATION, forbiddenException.getMessage())
```

Comment:
aha, what about. Just catching the forbidden exception and just try again with new cofig. And if forbidden exception is thrown again we will just throw it to the user instead of trying with new config. 🤔

Reply:
The solution you're proposing might work.